Mathlib/FieldTheory/AxGrothendieck.lean

```diff
@@ -152,13 +152,15 @@ theorem realize_genericPolyMapSurjOnOfInjOn
       S.InjOn f ↔ ∀ x y, x ∈ S → y ∈ S → f x = f y → x = y := by
     simp [Set.InjOn]; tauto
   simp only [Sentence.Realize, Formula.Realize, genericPolyMapSurjOnOfInjOn, Formula.relabel,
```

Comment:
(Oof, this is way too huge of a `simp only`. That goes on my tech debt todo list.)